docs/marionette.region.md

```diff
@@ -381,98 +308,3 @@ var MyView = Mn.View.extend({
   }
 });
 ```
-
-## Region Events And Callbacks
-
-A region will raise a few events on itself and on the target view when showing and destroying views.
-
-### Events Raised on the Region During `show()`
-
-* `before:show` / `onBeforeShow` - Called after the view has been rendered, but before its been displayed.
-* `show` / `onShow` - Called when the view has been rendered and displayed.
-* `before:empty` / `onBeforeEmpty` - Called before the view has been emptied.
-* `empty` / `onEmpty` - Called when the view has been emptied.
-
-```javascript
-view.supportsRenderLifecycle = true;
-view.supportsDestroyLifecycle = true;
-```
-
-## Custom Region Classes
-
-You can define a custom region by extending from
-`Region`. This allows you to create new functionality,
-or provide a base set of functionality for your app.
-
-### Attaching Custom Region Classes
-
-Once you define a region class, you can attach the
-new region class by specifying the region class as the
-value. In this case, `addRegions` expects the constructor itself, not an instance.
-
-```javascript
-var Mn = require('backbone.marionette');
-
-var FooterRegion = Mn.Region.extend({
-  el: "#footer"
-});
-
-var MyView = Mn.View.extend({
-  regions: {
-    footerRegion: FooterRegion
-  }
-});
-```
-
-You can also specify a selector for the region by using
-an object literal for the configuration.
-
-```javascript
-var Mn = require('backbone.marionette');
-
-var FooterRegion = Mn.Region.extend({
-  el: "#footer"
-});
-
-var MyView = Mn.View.extend({
-  regions: {
-    footerRegion: {
-      regionClass: FooterRegion
-      el: "#footer",
-    }
-  }
-});
-```
-
-Note that a region must have an element to attach itself to. If you
-do not specify a selector when attaching the region instance to your
-Application or LayoutView, the region must provide an `el` either in its
-definition or constructor options.
-
-### Instantiate Your Own Region
```

Comment:
Yeah I think that is a good graphic.   I mean you could do anything a custom region would do with a behavior, but it seems like in some cases it would be more complex..  But then you can do everything with a plain view and mixin in functionality..  that you can do with a view and a mn.behavior too..  I think there are some valid use cases for keeping both though.